.vscode/launch.json

```diff
@@ -10,6 +10,7 @@
       "request": "launch",
       "program": "${workspaceFolder}/InvenTree/manage.py",
       "args": ["runserver"],
+      "pythonArgs": ["-Xfrozen_modules=off"],
```

Comment:
Question: Why is this added?

Reply:
Trying to remove an annoying warning in debug console. This did not fix, I will remove it.